roles/confluent.ssl/tasks/main.yml

```diff
@@ -32,10 +32,13 @@
   include_tasks: provided_keystore_and_truststore.yml
   when: ssl_provided_keystore_and_truststore|bool
 
-- set_fact:
-    certs_updated: true
-
 - name: Delete SSL Certificate Generation Directory
   file:
     path: /var/ssl/private/generation
     state: absent
+
+- name: Set truststore and keystore file permissions
+  command: find /var/ssl/private/ -type f -exec chmod 0644 {} \;
```

Comment:
We should probably use the "file" module for this one: https://docs.ansible.com/ansible/latest/modules/file_module.html

Something like:
```
- name: Set Truststore and Keystore File Permissions
  file:
    path: "{{item}}"
    mode: '0644'
  loop:
    - "{{keystore_path}}"
    - "{{truststore_path}}"
```

There can also be a group/owner set in that same module... its a little tricky to say what to set those to

Reply:
Looks like you're getting this added in #185 so we should be good to close this out.

Thanks.